File: cs50x/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height of pyramid: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        // Add space at right
        for (int b = height; b > i + 1; b--)
        {
            printf(" ");
        }

        // Add right side of pyramid
        for (int a = 0; a <= i; a++)
        {
            printf("#");
        }
        printf("  ");

        // Add left side of pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50x/sentimental-mario-more/mario.py
from cs50 import get_int

# TODO

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    for s in range(height, i + 1, -1):
        print(" ", end="")

    for r in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for l in range(i + 1):
        print("#", end="")

    print()